python/tests/test_topology.py

```diff
@@ -97,6 +98,618 @@ def generate_segments(n, sequence_length=100, seed=None):
     return segs
 
 
+def kc_distance_tree(tree1, tree2, lambda_param=0):
+    """
+    Returns the Kendall-Colijn distance between the specified pair of trees.
+    lambda_param determines weight of topology vs branch lengths in calculating
+    the distance. Set lambda_param at 0 to only consider topology, set at 1 to
+    only consider branch lengths. See Kendall & Colijn (2016):
+    https://academic.oup.com/mbe/article/33/10/2735/2925548
+    """
+    samples = tree1.tree_sequence.samples()
+    if not np.array_equal(samples, tree2.tree_sequence.samples()):
```

Comment:
Does .samples() give sorted samples by default? If not, we may need to sort them. If yes, I'm guessing this is why you didn't write `tree1.samples()`, maybe you could add a short comment that we want to get the samples via the tree_sequence so that things are sorted.

Reply:
``.samples`` returns the samples in sorted order, yes.